net-misc/knock/knock-0.7-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Simple port-knocking daemon"
+HOMEPAGE="https://www.zeroflux.org/projects/knock
+	https://github.com/jvinet/knock"
+SRC_URI="https://www.zeroflux.org/proj/knock/files/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+IUSE="+server"
+
+DEPEND="server? ( net-libs/libpcap )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	sed -e "/^AM_CFLAGS/s: -g : :" \
+		-e "/dist_doc_DATA/s:COPYING::" \
+		-i Makefile.in || die
+	sed -e "s:/usr/sbin/iptables:/sbin/iptables:g" \
+		-i knockd.conf || die
+
+	default
```

Comment:
I recommend to place this before all modifications so that patches are guaranteed to apply over (vanilla) sources.